### DISCUSSION TITLE
Uno default temp. (WinUI .Net7.0/8.0) CommunityToolkit.WinUI v8

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#14635] [Q&A] [answered]
**How to get the WrapPanel and SaggeredPanel and if possible everything else. truly work with Uno?**

Dear **Legends,**

I have created a help.sln uno proj. and ask **4 question** on MainPage.xaml and ask for Uno.Youtube.CodeALonges videos. i don't know about any body else but **UI** is very time consuming, if not totaly impossible with **Uno** hihi.. 

I think i am not the only one with these questions concerning CommunityToolkit!

Thank you for reading and reactions,
D

VS version: 17.8.0 preview 6.0

**Comment by Daantje007** [comment]
https://www.mediafire.com/file/i5nykp38tbe84uu/Help2_%25282%2529.zip/file

**Comment by jeromelaban** [comment]
I'm not sure to understand your question, but you can use this sample to get the community toolkit working in your app: https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui.

**Comment by Daantje007** [comment]
Hi Jerome, 

I tried to create a new uno WinUI v.5 .Net8.0 template on the newest visual studio.  and just did exactly as in your document. but it is not working. 

I have added the packages v 7.1.2xx  uno.toolk.. to the projects and microsoft.toolk.. equivalent to the winsows project.
I am not entirely sure what to do with the: **shared proj.** and the **base proj.** as it is not documented. Tried with and without the package there.

    
![image](https://github.com/unoplatform/uno/assets/95089394/d7545e9f-b4fe-423a-b666-089d5f7f8ce7)
![image](https://github.com/unoplatform/uno/assets/95089394/f7554548-17e5-4fa6-a690-b538357783f4)


Also i have read: 
[here](https://devblogs.microsoft.com/ifdef-windows/announcing-windows-community-toolkit-v8-0/)
which you should only apply on the shared (Vit (below in Net8.0)) or so. only works on **windows proj.**

its probably my mistake but I need to get used to the new structure.

**Reply by jeromelaban** [reply, marked as answer]
Thanks, you can view a working sample here: https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoWCTDataGridSample

**Reply by Daantje007** [reply]
Hi Jerome,

You May delete this discussion. 

Thank you for Reading,
D
